imap/pop3d.c

```diff
@@ -99,6 +99,10 @@
 #define MAXHOSTNAMELEN 256
 #endif
 
+#ifndef MAXHOSTNAMELEN
+#define MAXHOSTNAMELEN
```

Comment:
In the original patch that we already have, this is defined to `256`, but in this version, it's defined without a value.  Do you know what the difference is?

I also just realised, the "256" definition is immediately above, so it will already be defined and not do anything.